cmd/podman/images/build_prune.go

```diff
@@ -0,0 +1,94 @@
+package images
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/podman/v3/cmd/podman/common"
+	"github.com/containers/podman/v3/cmd/podman/registry"
+	"github.com/containers/podman/v3/cmd/podman/utils"
+	"github.com/containers/podman/v3/cmd/podman/validate"
+	"github.com/containers/podman/v3/pkg/domain/entities"
+	"github.com/containers/podman/v3/pkg/specgenutil"
+	"github.com/spf13/cobra"
+)
+
+var (
```

Comment:
This looks like a nearly verbatim copy of `cmd/podman/images/prune.go`.  I'd much prefer to duplicate code.  Could you investigate using an alias? cobra supports that.

Reply:
@vrothberg Thanks :) just needed to add extra parents to `image/prune.go`

Reply:
Great work, thank you!